src/realm/object-store/sync/sync_session.cpp

```diff
@@ -47,6 +49,23 @@ using SessionWaiterPointer = void (sync::Session::*)(util::UniqueFunction<void(s
 constexpr const char SyncError::c_original_file_path_key[];
 constexpr const char SyncError::c_recovery_file_path_key[];
 
+namespace realm {
+
+std::ostream& operator<<(std::ostream& out, const SyncSession::ConnectionState& val)
```

Comment:
it doesn't look like this is used anywhere. do we need it?

Reply:
No, probably not - this was left over from when I was using the connection state to detect the reconnect, but that has been replaced by the error check in event hook. I'll remove it.